ipywidgets/static/widgets/js/manager.js

```diff
@@ -332,19 +348,19 @@ define([
          *      widget_class: (optional) string
          *          Target name of the widget in the back-end.
          *      comm: (optional) Comm
+         *           Comm object associated with the widget.
+         *      model_id: (optional) string
+         *           If not provided, the comm id is used.
          *
-         * Create a comm if it wasn't provided.
+         * Either a comm or a model_id must be provided.
```

Comment:
create_model does not need a comm or a comm_id.

`new_model` is meant for 
 - creating models for a given comm (in the context of persistence)
 - creating comm-less models (for dead comm widgets with broken links)
In the latter case, a model id is required.